developerportal/apps/ingestion/tests/test_data/blog.mozvr.com_rss.xml

```diff
@@ -0,0 +1,506 @@
vice?</p><p>Reading the Oculus <a href="https://www.oculus.com/legal/privacy-policy/" rel="noreferrer nofollow noopener">privacy policy</a>, they say that they process and collect information like</p><ul><li>information about your environment, physical movements, and dimensions</li><li>location-related information</li><li>information about people, games, content, apps, features, and experiences you interact with</li><li>identifiers that may be unique to you</li><li>and much much more!</li></ul><p>That’s…a lot of data. Most of this data, like processing information about your physical movements is required for basic functionality of most MR experiences. For example, to track whether you avoid an obstacle in BeatSaber, your device needs to know the position of your head in space.</p><p>There’s a difference between <em>processing</em> and <em>collecting</em>. Like we mentioned, you can’t do much without processing certain data. Processing can either happen on the device itself, or on remote servers. Collecting data implies that it is stored remotely for a time period beyond what’s necessary for simply processing it.</p><p>Mozilla’s brand promise to our users is focused on security and privacy. So, while testing the Oculus Quest for Mozilla Mixed Reality products, we needed to know what kind of data was being sent to and from the device during a browsing session. The device has a developer mode that allows you to access advanced features by connecting it to your computer and using <a href="https://developer.android.com/studio/command-line/adb">Android Debug Bridge</a> (adb). From there, we used the developer mode and `adb` to install a custom trusted root certificate. This allows us to inspect the connections in depth.</p><p>So, what <em>is</em> Facebook transmitting from your device back to Facebook servers during a routine browsing session? From the data we saw, they’re reporting configuration and telemetry data, such as information about how long it took to fetch resources. For example, here’s a graph of the amount of data sent over time from the Oculus VR headset back to Facebook.</p><!--kg-card-begin: image--><figure class="kg-card kg-image-card kg-card-hascaption"><img src="https://blog.mozvr.com/content/images/2019/12/Screenshot-2019-12-03-14.06.51-1.png" class="kg-image" alt="How much is that new VR headset really sharing about you?"><figcaption>Bytes sent to Facebook IPs over time</figcaption></figure><!--kg-card-end: image--><p>The data is identified by both an <code>id</code>, which is consistent across browsing sessions, and a <code>session_id</code>. The <code>id</code> appears to be linked to the device hardware, because linking a Facebook account didn’t change the identifier (or any other information as far as we detected).</p><p>In addition to general timing information, Facebook also receives reports on more granular, URL level timing information that uses a unique URL ID.</p><!--kg-card-begin: code--><pre><code>"time_to_fetch": "1",
```

Comment:
This can be ignored - it's a saved version of a Mozilla VR RSS feed from new year